doc/faq.rst

```diff
@@ -281,3 +281,24 @@ Troubleshooting
       conda install spyder-kernels=<VERSION>
 
    For more information, go to the :ref:`starting-kernel-problems-ref` section in the :doc:`common-illnesses` page of our troubleshooting guide.
+
+
+.. dropdown:: Q: Spyder doesn't launch or is slow on macOS 11 Big Sur. How can I get it working?
+
+   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released in version 4.2.1, due out before the end of 2020.
```

Comment:
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released in version 4.2.1, before the end of 2020.
```

Reply:
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be part of version 4.2.1, to be released before the end of 2020.
```

Reply:
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released by the end of 2020 as part of version 4.2.1.
```

Started with Carlos' version to avoid the grammar/syntax problems with the other and reversed the order of the clauses, to avoid the complex/choppy phrasing present from my original version.